From: Benjamin Kaduk <kaduk@mit.edu>
Date: Sat, 12 Mar 2022 20:57:11 -0800
Subject: Disable rx/perf test

It hardcodes a port number on which to run the testing service,
which may conflict with some buildd services and result in failing
autotests.

Change-Id: I608c6a1a876e6a1e7309bc621b6ed60e1b63b97a
---
 tests/TESTS | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/TESTS b/tests/TESTS
index 07277e1..90fe0ff 100644
--- a/tests/TESTS
+++ b/tests/TESTS
@@ -16,7 +16,6 @@ opr/uuid
 ptserver/pt_util
 ptserver/pts-man
 rx/event
-rx/perf
 volser/vos-man
 bucoord/backup-man
 kauth/kas-man
